### PR TITLE
Documentation Update

### LINKED ISSUE #351: unauthorized_client An unauthorized client tried to access your resources.
I keep getting this issue:
`unauthorized_client An unauthorized client tried to access your resources.` 

When I go login to Otto it redirects me to the LMS login which redirects me to:
`http://127.0.0.1:8000/oauth2/authorize/confirm`
where I get that error.

### PR DESCRIPTION
**Pull request by clintonb**
- Copied requirements and basic documentation from the IDA template
- Testing docs build in Travis

Fixes #351 

@vkaracic the issue you are seeing in #351 usually indicates invalid OAuth 2.0 configuration. Check your client ID and secret. This PR includes some updated documentation.

FYI @rlucioni @jimabramson @bderusha @peter-fogg

### CODE REVIEW

**Comment by peter-fogg** [comment]
:+1: Looks awesome.

File: docs/Makefile [hunk 1]
```diff
@@ -77,17 +87,25 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/edXE-Commerce.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/cookiecutterproject_name.qhcp"
```
Comment (rlucioni) [inline docs/Makefile]: Why cookiecutterproject_name?

Comment (clintonb) [inline docs/Makefile]: Fixed.

**Comment by rlucioni** [comment]
Thanks for giving us a start on this @clintonb. :+1:

**Comment by bderusha** [comment]
:+1:

**Pull request merged**